src/Function.cpp

```diff
@@ -562,6 +570,43 @@ Function::OutputForwardDecl(std::ostream &out)
 	OutputHeader(out);
 	if(func_attr_inline)
 		out << " __attribute__((always_inline))";
+	if(func_attr_aligned){
+		int probability__BIGGEST_ALIGNMENT__ = 38;
```

Comment:
Why have you chosen value of `38`, I can imagine a bigger value (e.g. 4096, or even a bit bigger)?

Reply:
I have chosen random value.
rnd_flipcoin() accepts value from 0 to 100. It will not accept value > 100.

Reply:
I mean a bigger value for `probability__BIGGEST_ALIGNMENT__`, not for `rnd_flipcoin`.

Reply:
If I set bigger value for **probability__BIGGEST_ALIGNMENT__** greater than 100, it throws error stating **rnd_flipcoin** accepts value between [0-100]

rnd_flipcoin returns true or false based on probability passed as parameter to it. Whether to use biggest alignment or not, I have set probability = 38 and is passed to rnd_flipcoin() which will return true or false.

Should I increase probability value = 38 upto 100?

Reply:
Ah, sorry, I was confused with the difference in between `__BIGGEST_ALIGNMENT__` and `probability__BIGGEST_ALIGNMENT__`. So yes, probability of `38` is fine ;)